### PR TITLE
Add Hub manifest

### PR DESCRIPTION
**Pull request by robert-uhl**

### CODE REVIEW

**Review by chainlink — CHANGES_REQUESTED** [review]
Couple small things

**Comment by chainlink** [comment]
Did a diff against the existing pach template in hub and found the following:

- [ ] Dash Deployment shouldn't set limits
- [ ] etcd deployment shouldn't set limits
- [ ] Loki Logging should be true
- [ ] No expose docker socket should be false
- [ ] Pachd Service should not be nodeport
- [ ] tls-secret is empty (and shouldn't be included)

File: test/hub-values.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+dash:
```
Comment (chainlink) [inline test/hub-values.yaml:1]: Related to my comment below, but would be nice to eliminate the need for this file as it's the same as `examples/hub-values.yaml` minus the templating differences

File: go.sum [hunk 2]
```diff
@@ -64,6 +64,12 @@ github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534/go.mod h1:iroGtC8B3tQiqtds1l+mgk/BBOrxbqjH+eUfFQYRc14=
 github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd/go.mod h1:64YHyfSL2R96J44Nlwm39UHepQbyR5q10x7iYa1ks2E=
+github.com/Masterminds/goutils v1.1.1 h1:5nUrii3FMTL5diU80unEVvNevw1nH4+ZV4DSLVJLSYI=
+github.com/Masterminds/goutils v1.1.1/go.mod h1:8cTjp+g8YejhMuvIA5y2vz3BpJxksy863GQaJW2MFNU=
+github.com/Masterminds/semver v1.5.0 h1:H65muMkzWKEuNDnfl9d70GUjFniHKHRbFPGBuZ3QEww=
+github.com/Masterminds/semver v1.5.0/go.mod h1:MB6lktGJrhw8PrUyiEoblNEGEQ+RzHPF078ddwwvV3Y=
+github.com/Masterminds/sprig v2.22.0+incompatible h1:z4yfnGrZ7netVz+0EDJ0Wi+5VZCSYp4Z0m2dk6cEM60=
```
Comment (chainlink) [inline go.sum:71]: Are the go.sum/mod changes still required? (looks like we don't need sprig anymore, probably the other stuff too?

File: pachyderm/templates/pachd/tls-secret.yaml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 apiVersion: v1
 data:
-  tls.crt: {{ .Values.tls.crt | b64enc | quote }}
-  tls.key: {{ .Values.tls.key | b64enc | quote }}
+  tls.crt: {{ .Values.tls.create.crt | b64enc | quote }}
+  tls.key: {{ .Values.tls.create.key | b64enc | quote }}
 kind: Secret
 metadata:
   labels:
-    app: pachd-tls-cert
```
Comment (chainlink) [inline pachyderm/templates/pachd/tls-secret.yaml:13]: "App" should actually be pachyderm (labels are generally used to isolate installs of a given application on a cluster)

File: pachyderm/templates/dash/ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.dash.url -}}
+apiVersion: "networking.k8s.io/v1beta1"
+kind: "Ingress"
+metadata:
+  name: "dash"
+  annotations:
```
Comment (chainlink) [inline pachyderm/templates/dash/ingress.yaml:6]: Can you please add a labels section here:

```
  labels:
    app: dash
    suite: pachyderm
```

**Review by chainlink — CHANGES_REQUESTED** [review]
Couple more changes

File: examples/hub-values.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  resources:
+    requests:
+      cpu: "1"
+      memory: 2G
+    limits: null
+  service:
+    annotations:
+      prometheus.io/port: "2379"
+      prometheus.io/scrape: "true"
+  storageClass: "ssd-storage-class"
```
Comment (chainlink) [inline examples/hub-values.yaml:18]: By setting this here, no storage class object gets created.

**Review by chainlink — CHANGES_REQUESTED** [review]

File: examples/hub-values.yaml (hunk 5 above)
Comment (robert-uhl) [reply examples/hub-values.yaml:18]: Yup, in Hub that is performed by a different step.

**Pull request merged**